internal/services/ticker/ticker.go

```diff
@@ -115,6 +128,10 @@ func New(fs ...TickerOpt) (*TickerImpl, error) {
 		return nil, fmt.Errorf("tenant alerter is required. use WithTenantAlerter")
 	}
 
+	if opts.partitionId == "" {
```

Comment:
You're not setting this opt in the `TickerImpl`

Reply:
```
func WithPartitionId(pid string) TickerOpt {
	return func(opts *TickerOpts) {
		opts.partitionId = pid
	}
}
```

I am

Reply:
That's in the `TickerOpts`, but unless I'm missing something it's not set in the TickerImpl